services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/PageableQuery.java

```diff
@@ -97,6 +123,24 @@ default Page<Document> findDocumentsWithPagination(
         return new PageImpl<>(data, pageable, totalElements);
     }
 
+    default Page<Document> findDocuments(
+            @Nonnull MongoTemplate mongoTemplate,
+            @Nonnull String collectionName,
+            @Nonnull int limit,
+            @Nonnull Criteria criteria,
+            @Nonnull Sort sort,
+            @Nullable List<String> excludedFields) {
+        List<String> fieldsToExclude = excludedFields != null ? excludedFields : Collections.emptyList();
+        Query mongoQuery = Query.query(criteria).with(sort).limit(limit);
+        if (!fieldsToExclude.isEmpty()) {
+            for (String field : fieldsToExclude) {
+                mongoQuery.fields().exclude(field);
+            }
+        }
+        List<Document> results = mongoTemplate.find(mongoQuery, Document.class, collectionName);
+        return new PageImpl<>(results, PageRequest.of(0, limit), results.size());
```

Comment:
chore: instead of duplicating logic, just reuse the generic implementation and pass in the Document class.
```suggestion
       return findGeneric(mongoTemplate, collectionName, limit, criteria, sort, excludedFields, Document.class);
```